src/Navigation/NavBar/index.css

```diff
@@ -10,4 +10,18 @@
 button {
   color: var(--primary-color);
   background-color: var(--button-color);
+  width: 85px;
+  height: 25px;
+  font-size: 15px;
+  border-radius: 100px;
+  font-weight: 400;
+  outline: none;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
Colors in css variables girl!
It would be easier if you need to change the theme in future.